CHANGELOG.MD

```diff
@@ -1,4 +1,95 @@
-﻿## [1.17.1](https://github.com/PowerShell/PSScriptAnalyzer/tree/1.17.1) - 2018-06-06
+﻿# CHANGELOG
+
+## [1.18.0](https://github.com/PowerShell/PSScriptAnalyzer/tree/1.18.0) - 2019-03-??
+
+### Breaking Changes
+
+- Minimum required version of `PowerShell Core` was raised from `6.0.2` to `6.1.0` due to `6.0` having reached the end of it's support life-cycle. (#1165)
+- Updated used version of `Newtonsoft.Json` to `11.0.2`. (#1165) (by @bergmeister)
+- Command data files of PowerShell `6.0.2` were removed. Updated files for PowerShell `6.1.0` were added. This means that users that have customized the `UseCompatibleCmdlets` rule, need to either adapt the version string as per rule documentation or can alternatively copy in the old command data files into the `Settings` installation folder (#1075) (by @bergmeister)
```

Comment:
Is this `6.1.0` or `6.1.3`?

Reply:
6.1.0 because we did the PR early on. But since PowerShell patches are mostly patches of the .Net Core Runtime I don't think cmdlets or their parameters were added/renamed/removed unless you are aware of something

Reply:
No, there won't be any serious changes in PowerShell itself. It might be worth just calling this `PowerShell 6.1`.

Reply:
Ok. Good point. The docs of the rule already state what I said in my comment, therefore I agree to shorten it to just 6.0 keep the changelog notes rather simple